pkg/ccl/changefeedccl/checkpoint/checkpoint_test.go

```diff
@@ -175,105 +173,58 @@ func TestCheckpointRestore(t *testing.T) {
 	}
 
 	for name, tc := range map[string]struct {
-		initialHighWater       hlc.Timestamp
-		oldCheckpointTs        hlc.Timestamp
-		oldCheckpointSpans     []roachpb.Span
-		newCheckpoint          *jobspb.TimestampSpansMap
-		expectedError          error
-		expectedFrontierForOld checkpointSpans
-		expectedFrontierForNew checkpointSpans
+		trackedSpans      roachpb.Spans
+		initialHighWater  hlc.Timestamp
+		savedCheckpoint   *jobspb.TimestampSpansMap
+		checkpointedSpans checkpointSpans
+		expectedError     string
```

Comment:
i'm a little confused as to which of these fields are the test input and which are the expectation. can you make that a bit clearer?